doc/source/news/14.md

```diff
@@ -1,5 +1,86 @@
 # News - 14 series
 
+(release-14-0-4)=
+## Release 14.0.4 - 2024-05-29
+
+### Fixes
+
+* [{doc}`/reference/functions/query_parallel_or`] Fixed a bug where the `match_escalation_threshold` or `force_match_escalation` options were ignored when using `query_parallel_or`.
+
+  * For example, before the fix, even when `match_escalation_threshold` was set to disable match escalation, the results still escalated.
+
+  ```
+  #$GRN_QUERY_PARALLEL_OR_N_CONDITIONS_THRESHOLD=1
+  #$GRN_QUERY_PARALLEL_OR_N_THREADS_LIMIT=2
+
+  table_create Data TABLE_NO_KEY
+  column_create Data content COLUMN_SCALAR ShortText
+
+  load --table Data
+  [
+  {"content": "hello world"},
+  {"content": "It's an apple"}
+  ]
+
+  table_create Terms TABLE_PAT_KEY ShortText   --normalizers NormalizerNFKC150   --default_tokenizer TokenNgram
+  column_create Terms data_content COLUMN_INDEX|WITH_POSITION Data content
+  select Data   --filter 'query_parallel_or(["content"], "a")'   --match_escalation_threshold -1
+  [
+    [
+      0,
+      0.0,
+      0.0
+    ],
+    [
+      [
+        [
+          1
+        ],
+        [
+          [
+            "_id",
+            "UInt32"
+          ],
+          [
+            "content",
+            "ShortText"
+          ]
+        ],
+        [
+          2,
+          "It's an apple"
+        ]
+      ]
+    ]
+  ]
+  ```
+
+* Fixed a bug where full-text search on a reference column of a vector didn't work.
+
+  * For exampe, before the fix, Groonga always returned 0 hits when performing full-text search on a reference column of a vector. Internally, this happened because Groonga couldn't correctly retrieve `key_type` for reference columns.
```

Comment:
```suggestion
  For exampe, before the fix, Groonga always returned 0 hits when performing full-text search on a reference column of a vector. Internally, this happened because Groonga couldn't correctly retrieve `key_type` for reference columns.
```